src/entities/paginated_collection.coffee

```diff
@@ -137,7 +137,7 @@ define [], ->
     #
     # @param data [Object] the response returned from server
       parse: (data) ->
-        if Backbone.Collection.prototype.parse != collection.parse
+        if collection.parse? && (Backbone.Collection.prototype.parse != collection.parse)
```

Comment:
Does it matter? If parse is there and it is not Backbone.Collection.prototype.parse, then it is false. If collection.parse is null, undefined, or some other falsy value then it is not Backbone.Collection.prototype.parse, and it is false.